src/components/Badge/styled/container.js

```diff
@@ -32,14 +36,14 @@ const StyledContainer = styled.span`
         props.variant === 'brand' &&
         `
             color: ${COLOR_WHITE};
```

Comment:
Here we need to get the contrast text computed as we did in the Button component. This is necessary because if the customer sets a light brand color we need to present a text color with the right contrast text in order to make it visually possible. One clear example of this if when the user sets #fff color as a brand, keeping text color to #fff as well will be visually impossible since all we get is a white box.